### DISCUSSION TITLE
State of the helm chart?

### DISCUSSION THREAD

**Discussion by xeruf** [toeverything/AFFiNE#6325] [Q&A]
There was a helm chart repo at https://github.com/toeverything/helm-charts but it seems abandoned and missing files.
Would like to deploy affine on kubernetes so what is the state there?

**Comment by Brooooooklyn** [comment]
https://github.com/toeverything/AFFiNE/tree/canary/.github/helm/affine

**Reply by xeruf** [reply]
Thanks! Where is the helmrepo to pull the chart? I am not familiar how github handles this.

**Reply by nikatar** [reply]
It doesn't work. From non-existing image tags to missing dependencies for redis and pg

**Reply by mfreeman451** [reply]
I've got it to work, it was a bit of work/hacking, its not pretty but I can share it if you really want it.

**Reply by maximveksler** [reply]
@mfreeman451 let's get it to a working condition
